src/helpers/redirects.ts

```diff
@@ -65,6 +71,7 @@ export const generateStaticRedirects = ({
   }
 }
 
+// eslint-disable-next-line max-lines-per-function
```

Comment:
[sand] This function is indeed starting to get pretty complex. Could you split it out?

Reply:
I'\ve split out the various bits